Source/WebCore/platform/mac/ScrollbarThemeMac.mm

```diff
@@ -582,11 +583,14 @@ static void paintScrollbar(Scrollbar& scrollbar, GraphicsContext& context)
     return true;
 }
 
-void ScrollbarThemeMac::paintScrollCorner(ScrollableArea&, GraphicsContext& context, const IntRect& cornerRect)
+void ScrollbarThemeMac::paintScrollCorner(ScrollableArea& area, GraphicsContext& context, const IntRect& cornerRect)
 {
     if (context.paintingDisabled())
         return;
 
+    // Keep this in sync with ScrollAnimatorMac's effectiveAppearanceForScrollerImp:.
+    LocalDefaultSystemAppearance localAppearance(area.useDarkAppearance());
```

Comment:
This should be `useDarkAppearanceForScrollbars()`.

Reply:
Apologies I didn't see that on the type I looked at but I was looking at the wrong type.

Reply:
Fixed. Sorry for that.